codecov.yaml

```diff
@@ -0,0 +1,26 @@
+codecov:
```

Comment:
Could you briefly explain the meaning of each parameter?

Reply:
L1-L20 is default value. I copied from [here](https://docs.codecov.com/docs/codecov-yaml#default-yaml)
L22-26:
https://github.com/autowarefoundation/autoware.universe/pull/384/files#diff-dd0aaed42efd4c8adb46abdee691cfb7a3f4a56cfb45caef82da9b3e9b5469b2R22-R26
This is a setting to enable the carry forward flag.
I have set the flag name `total`. This flag will cause the code coverage for all paths in this repository to inherit the previous coverage results if they have not changed. [Reference](https://docs.codecov.com/docs/carryforward-flags#how-carryforward-flags-work-a-motivating-example)

Reply:
I think if you don't override them, we can remove them.
Would you like to add them?

Reply:
https://docs.codecov.com/docs/codecovyml-reference
Some setting has default value. So I removed them.